vendor/engines/secure_rooms/lib/secure_rooms/engine.rb

```diff
@@ -43,12 +43,8 @@ class Engine < ::Rails::Engine
 
       ViewHook.add_hook "search.results_table", "extra_headers", "search/extra_headers"
       ViewHook.add_hook "search.results_table", "extra_columns", "search/extra_columns"
-    end
 
-    initializer "secure_rooms.action_controller" do
-      ActiveSupport.on_load :action_controller do
-        helper SecureRooms::SecureRoomsHelper
-      end
+      ApplicationController.helper SecureRooms::SecureRoomsHelper
```

Comment:
We should make a note of how to test this on NU stage

Reply:
https://nucore-staging.northwestern.edu/facilities/all/users/23782/card_number/edit